.github/workflows/any-branch-uploads.yml

```diff
@@ -16,6 +16,7 @@ jobs:
         docker run -v `pwd`/docs/v1:/usr/src/p4-spec p4lang/p4rt-madoko:latest make
         ls docs/v1/build
     - name: Upload spec to S3 if needed
+      if: ${{ github.actor != 'dependabot[bot]' }}
```

Comment:
I think you could condition the whole job instead of just this step:
```
if: ${{ github.repository == 'p4lang/p4runtime' && github.actor != 'dependabot[bot]' }}
```

Reply:
I thought of that, but then decided it wouldn't hurt to do the build part and just skip the publishing, to get more "coverage" (although it might be redundant with other workflows).  What do you think?

Reply:
I don't think strongly about it. Dependabot PRs should have no impact on spec generation, but it's not like we have a high volume of such PRs anyway.